README.md

```diff
@@ -1,15 +1,50 @@
 # Mink WebDriver Classic Driver
 
+[![PHP Version](http://poser.pugx.org/mink/webdriver-classic-driver/require/php)](https://packagist.org/packages/mink/webdriver-classic-driver)
 [![Latest Stable Version](https://poser.pugx.org/mink/webdriver-classic-driver/v)](https://packagist.org/packages/mink/webdriver-classic-driver)
 [![Latest Unstable Version](https://poser.pugx.org/mink/webdriver-classic-driver/v/unstable)](https://packagist.org/packages/mink/webdriver-classic-driver)
 [![Total Downloads](https://poser.pugx.org/mink/webdriver-classic-driver/downloads)](https://packagist.org/packages/mink/webdriver-classic-driver)
 [![CI](https://github.com/minkphp/webdriver-classic-driver/actions/workflows/ci.yml/badge.svg)](https://github.com/minkphp/webdriver-classic-driver/actions/workflows/ci.yml)
 [![License](https://poser.pugx.org/mink/webdriver-classic-driver/license)](https://github.com/minkphp/webdriver-classic-driver/blob/main/LICENSE)
 [![codecov](https://codecov.io/gh/minkphp/webdriver-classic-driver/branch/main/graph/badge.svg?token=11hgqXqod9)](https://codecov.io/gh/minkphp/webdriver-classic-driver)
 
-## Usage Example
+A [php-webdriver](https://github.com/php-webdriver/php-webdriver)-based driver
+for [Mink](https://github.com/minkphp/Mink), supporting Selenium 2-4 and tested on Chrome, Firefox and Edge.
 
-``` php
+## Installation
+
+### For Mink/Standalone Usage
+
+Simply install the package with composer and check the [Standalone Usage Example](#standalone-usage-example) below.
+
+```shell
+composer require "behat/mink" "mink/webdriver-classic-driver"
+```
+
+### For Behat/BDD Usage
+
+1. Add all the relevant packages - typically this will be for testing your application, so you'd want to use `--dev`.
+    ```shell
+    composer require --dev "behat/behat" "behat/mink" "friends-of-behat/mink-extension" "mink/webdriver-classic-driver"
+    ```
+2. Configure Behat to enable the Mink Extension and set it to use this driver, for example:
+    ```yaml
+    # behat.yml
+    default:
+        extensions:
+            Behat\MinkExtension\Extension:
+                browser_name: chrome
+                base_url: https://example.com/
+                sessions:
+                    default:
+                        webdriver-classic: ~
+    ```
+
+## Standalone Usage Example
+
+You'll need to set up Selenium. Refer to [Contributing \ Testing](.github/CONTRIBUTING.md#testing) to see how.
```

Comment:
@stof this isn't what we decided on earlier, but I was thinking that we should avoid huge README when possible, and in this case might as well link to the contrib file. Wdyt?

Reply:
I don't think this makes sense as is, especially given that the contributing instructions suggest `docker compose up` which won't work as is for projects using the driver.

I suggest

> You need to run a WebDriver server, either using Selenium or the browser driver directly.

With `Selenium` being a link to the Selenium website.

Reply:
Good point, the docker-compose file we provided is specifically for testing (and they won't actually see that in their project).